app/models/article.rb

```diff
@@ -28,10 +28,11 @@ class Article < ApplicationRecord
     validates :article_tags, presence: true
   end
 
-  scope :released, -> { where.not(shipped_at: nil) }
+  scope :shipped, -> { where.not(shipped_at: nil) }
+  scope :wip, -> { where(shipped_at: nil) }
 
-  def shipped!
-    self.shipped_at = Time.now
+  def ship
```

Comment:
![](http://images.uncyc.org/ja/thumb/9/96/%E3%81%8A%E3%81%BE%E3%81%88%E3%81%AF%E4%BD%95%E3%82%92%E8%A8%80%E3%81%A3%E3%81%A6%E3%81%84%E3%82%8B%E3%82%93%E3%81%A0.jpg/300px-%E3%81%8A%E3%81%BE%E3%81%88%E3%81%AF%E4%BD%95%E3%82%92%E8%A8%80%E3%81%A3%E3%81%A6%E3%81%84%E3%82%8B%E3%82%93%E3%81%A0.jpg)